core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/config/Flow2Config.java

```diff
@@ -21,10 +22,10 @@
     public Map<String, FlowConfiguration<?, ?>> flowConfigurationMap() {
         Map<String, FlowConfiguration<?, ?>> flowConfigMap = new HashMap<>();
         for (FlowConfiguration<?, ?> flowConfig : flowConfigs) {
-            for (FlowEvent event : flowConfig.getFlowTriggerEvents()) {
+            for (FlowEvent event : flowConfig.getEvents()) {
```

Comment:
what garanties that stringrepresentations of events in different event types won't clash